### DISCUSSION TITLE
Way to reset added codes?

### DISCUSSION THREAD

**Discussion by J43L35** [gopro/labs#159] [Q&A] [answered]
Thank you to everyone who has worked on these functionality extensions! I just started using labs on my two GoPro Hero 11s. Before I start experimenting more, I'd like to find out, if a code that has been permanently implemented and you'd like to reset it, is there a way?

I've seen that creating a QR with no value entry can effectively neutralize it, but I was wondering if it's possible to remove them completely. For example, if you set a LOGO and then want to remove it, you can leave it empty, but when the GoPro powers on it still says that LOGO is one of the settings added.

**Comment by dnewman-gpsw** [comment, marked as answer]
Very simple, !RESET is the command you want. This will only reset any permanent Labs features, your camera normal settings with not be affected, and media is untouched.

![image](https://user-images.githubusercontent.com/10856084/206266637-78ac59bc-e2c1-4774-8deb-05a496c77743.png)

**Reply by J43L35** [reply]
Awesome, thank you! With that as an option, really looking forward to experimenting!!
